pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
     "click>=8.1.3", # for CLI interface
     "clr-loader==0.2.6",
     "grpcio>=1.30.0",
-    "protobuf>=3.12.2",
+    "protobuf>=3.12.2,<5",
```

Comment:
Interesting... Any reason why? There shouldn't be a limitation anymore on ansys-api-product-instancemanager

Reply:
Will update this once https://github.com/ansys/ansys-api-mechanical/pull/3 get merged. Because of `ansys-api-mechanical` restricts <5 I can still see the issue with python 3.9